guides/release/templates/binding-element-attributes.md

```diff
@@ -45,7 +45,7 @@ This means that some uncommon but perfectly valid attributes, such as `lang` or
 custom `data-*` attributes must be specifically enabled. For example, this template:
 
 ```handlebars
-{{#link-to "photos" data-toggle="dropdown" lang="es"}}Fotos{{/link-to}}
+<LinkTo @route="photos" data-toggle="dropdown" lang="es">Fotos</LinkTo>
```

Comment:
This section should be re-written. Specifically the guidance just below:

> To enable support for these attributes, an attribute binding must be added for each specific attribute on the component. To do this, you can extend the appropriate components...

Instead we should teach using attributes vs arguments.

Reply:
Let's do this work on a follow up PR so this work doesn't get blocked: https://github.com/ember-learn/guides-source/issues/911